<launch>
  
  <!-- Enable baxter robot -->
  <node name="enable_baxter_robot" pkg="baxter_tools" type="enable_robot.py" output="screen" args="-e"/>
  
  <!-- Joint controller -->
  <node name="priovr_joint_controller" pkg="baxter_teleop" type="priovr_joint_controller.py" output="screen">
  </node>
  
  <!-- PrioVR node -->
  <node name="get_joint_states" pkg="priovr" type="get_joint_states.py" output="screen">
    <rosparam command="load" file="$(find baxter_teleop)/config/priovr_params.yaml"/>
    <rosparam command="load" file="$(find priovr)/config/priovr_common.yaml"/>
  </node>
  
  <!-- Show in Rviz -->
  <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find baxter_teleop)/config/cartesian_controller.rviz"/>
  
</launch>
